libr/anal/sign.c

```diff
@@ -837,6 +837,229 @@ R_API bool r_sign_delete(RAnal *a, const char *name) {
 	return sdb_remove (a->sdb_zigns, k, 0);
 }
 
+static bool matchBytes(RSignItem *a, RSignItem *b) {
+	if (a->bytes && b->bytes) {
+		if (a->bytes->size == b->bytes->size) {
+			return !memcmp (a->bytes->bytes, b->bytes->bytes, b->bytes->size);
```

Comment:
bytes have binary mask, that wont work for all cases, but its fine for me to merge it for now

Reply:
yeah just copy pasta from radiff for now

Reply:
oh i see... will try to fix